src/test/java/com/google/api/codegen/protoannotations/testdata/library.proto

```diff
@@ -251,9 +251,6 @@ service LibraryService {
   // Test bidi-streaming.
   // gRPC streaming methods don't have an HTTP equivalent and don't need to have the google.api.http option.
   rpc DiscussBook(stream DiscussBookRequest) returns (stream Comment) {
-    option (google.api.method_signature) = {
```

Comment:
This method signature didn't exist in the library_gapic.yaml, so it's getting removed.